lib/wallet/client.js

```diff
@@ -63,6 +63,15 @@ class WalletClient extends NodeClient {
     return parseEntry(await super.getEntry(block));
   }
 
+  async getMedianTime(blockhash) {
+    if (Buffer.isBuffer(blockhash))
+      blockhash = util.revHex(blockhash);
+
+    const blockheader = await super.execute('getblockheader', [blockhash]);
```

Comment:
Is there any chance that the `blockheader` can be `null`?

Reply:
Yeah, there may be. I am not sure that current usage would reach that.

Reply:
It's a problem if there isn't a result in all current usage.